modules/zio-config/docs/README.md

```diff
@@ -0,0 +1,38 @@
+# ZIO Config module for PureConfig
+
+Support for providing instances of `ConfigCovert` given instances of ZIO Config `ConfigDescriptor`
+
+## Add pureconfig-zio-config to your project
+
+In addition to the [PureConfig core](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-zio-config" % "@VERSION@"
+```
+
+## Example
+
+`Person` already has a `ConfigDescriptor` instance.
```

Comment:
There seems to be a bit of context lacking here. Users may not understand right away why `Person` already has a `ConfigDescriptor` instance, particularly if they don't use automatic derivation or `zio-config-magnolia` in particular.

I'd rather not introduce any additional dependencies, neither here nor in tests. From [ZIO Config docs](https://zio.github.io/zio-config/docs/configdescriptor/configdescriptor_index#a-simple-example), creating a `ConfigDescriptor` manually seems rather straightforward and it would make the requirements for this module clearer. Would you mind defining it manually here?

If you really feel that using automatic derivation is important here and in tests, please improve this paragraph to explain that `DeriveConfigDescriptor` triggers automatic derivation, possibly linking to appropriate ZIO Config documentation.

Reply:
@ruippeixotog I didn't want users to bother about how existing `ConfigDescriptor`s was implemented but it is also true that users should not need to know how automatic derivation works to understand the example. I'm removing `zio-config-magnolia` from the test dependencies as well since we are removing it from docs.